### DISCUSSION TITLE
Data importer: save config files to server

### DISCUSSION THREAD

**Discussion by grayfrost90** [firefly-iii/firefly-iii#8423] [Firefly III] [answered]
I've been playing around with Firefly and the data importer for a couple of weeks now and really like it.

Would it be possible to save the importer config JSON files directly to the data importer to then be used later on - from a drop down perhaps? This would mean I could easily import from different devices without having to remember to save the config somewhere central

Thanks

Jack

**Comment by JC5** [comment, marked as answer]
https://github.com/firefly-iii/data-importer/blob/main/.env.example#L137

I realize that this isn't in the documentation and I'll add it asap.

**Reply by grayfrost90** [reply]
fantastic! just uploaded a config file and it all works - perfect - and thanks for the quick response
